docs/commands/webserver.md

```diff
@@ -29,6 +29,7 @@ python3 -m fastchat.serve.huggingface_api_worker --model-info-file ~/elo_results
 export OPENAI_API_KEY=
 export ANTHROPIC_API_KEY=
 export GCP_PROJECT_ID=
+export YANDEXGPT_API_KEY=
```

Comment:
this is an internal doc so we want to keep it minimal. could you remove this?

Reply:
Sure